releasenotes/notes/deprecate-older-tornado-versions-381d2f4e6c4c8288.yaml

```diff
@@ -0,0 +1,4 @@
+---
+deprecations:
+  - |
+    tornado: Deprecated support for Tornado versions older than v6.1. Use Tornado v6.1 or later.
```

Comment:
can we get a deprecation notice based on version in the integration?

or else, we should write this like a normal in-code deprecation warning, e.g.:

> Deprecated support for Tornado versions older than v6.1, support will be removed in v4.0.0

or.. something... just a nit, this is probably ok as-is

Reply:
Yup, just did this